source/frontend/src/features/properties/map/project/add/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { TextArea } from 'components/common/form';
+import styled from 'styled-components';
+
+export const MediumTextArea = styled(TextArea)`
+  textarea.form-control {
+    min-width: 80rem;
+    height: 7rem;
+    resize: none;
+  }
+`;
+
+export const LargeTextArea = styled(TextArea)`
+  textarea.form-control {
+    min-width: 80rem;
+    height: 18rem;
+    width: 80rem;
+    resize: none;
+  }
```

Comment:
This is odd. Usually, the width is controlled by the container. i.e. column or SectionField

Reply:
Add Lease uses a similar approach.

Reply:
The lease section of the site was one of the first developed so we have a lot of tech debt there